java/pom.xml

```diff
@@ -66,8 +80,6 @@
         </dependencies>
     </dependencyManagement>
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Seems like we can get on without specifying these.

Reply:
Because we use Spring Boot's default now, I guess.